cmd/manager.go

```diff
@@ -56,4 +56,6 @@ func init() {
 	managerCmd.PersistentFlags().StringVar(&sbm.SpecifyCollector, "specify-collector", os.Getenv("SUPPORT_BUNDLE_COLLECTOR"), "Execute specify collector script. e.g., longhorn")
 	managerCmd.PersistentFlags().StringSliceVar(&sbm.ExcludeResourceList, "exclude-resources", getEnvStringSlice("SUPPORT_BUNDLE_EXCLUDE_RESOURCES"), "List of resources to exclude. e.g., settings.harvesterhci.io,secrets")
 	managerCmd.PersistentFlags().StringSliceVar(&sbm.BundleCollectors, "extra-collectors", getEnvStringSlice("SUPPORT_BUNDLE_EXTRA_COLLECTORS"), "Get extra resource for the specific components e.g., harvester")
+	managerCmd.PersistentFlags().StringVar(&sbm.Description, "description", os.Getenv("SUPPORT_BUNDLE_DESCRIPTION"), "The support bundle description")
```

Comment:
Suggest making the default string to "", so it doesn't change the current behavior.

Reply:
But, if `os.Getenv("SUPPORT_BUNDLE_DESCRIPTION")` is empty, then `sbm.Description` will be empty. I'm not sure what you mean.

Reply:
BTW, you could see the last screenshot of downloaded metadata.yaml in [this comment of original issue post](https://github.com/harvester/harvester/issues/4630#issue-1948553180). In current situation, both fields are already empty.

Reply:
my bad, I thought "The support bundle description" was the default string. It looks OK to me now.